### PR TITLE
feat: inputs css added

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview refer following codesandbox link:
```https://codesandbox.io/s/inputs-crux-fqq8c?file=/index.html```

### CODE REVIEW

**Review by Asuraking1n — COMMENTED** [review]
You can use some padding in checkboxes, also add iframes.

**Comment by Manishsharma203** [comment]
> You can use some padding in checkboxes, also add iframes.

sure

**Comment by TusharJade** [comment]
Just a suggestion add some margin, padding and width to inputs other than that it looks good

**Comment by Manishsharma203** [comment]
> Just a suggestion add some margin, padding and width to inputs other than that it looks good

sure. Will add some utility classes for that

File: Components/inputs.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    background: var(--border-secondary);
+}
+
+.check-input{
+    height: 1.2rem;
+    width: 1.2rem;
+    appearance: none;
+    -webkit-appearance: none;
+    background: white;
+    border: 2px solid #77b255;
```
Comment (krgarima) [inline Components/inputs.css:48]: use CSS variable.

**Pull request merged**